tests/cypress.json

```diff
@@ -3,9 +3,11 @@
   "video": false,
   "defaultCommandTimeout": 5000,
   "responseTimeout": 4000,
-
   "fixturesFolder": "fixtures",
   "integrationFolder": "tests",
+  "pageLoadTimeout": 10000,
```

Comment:
I think we should adjust all the timeouts to our *expectations* to also test if the performance of Busola isn't below the expected threshold. A page loading for 9.5s isn't a success, don't you think?